src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SPECIFIED_CHARACTER = ",";
+
+    public void getStatistic(String fromFileName, String toFileName) throws RuntimeException {
+        File fromFile = new File(fromFileName);
+        File toFile = new File(toFileName);
+        String statisticData;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFile))) {
+            statisticData = readStatisticData(reader);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file: " + fromFileName, e);
+        }
+
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFile))) {
+            writeStatisticData(writer, statisticData);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file: " + toFileName, e);
+        }
```

Comment:
Divide logic of reading and writing for separate private methods and return result as String